ansible/inventory.yml

```diff
@@ -268,9 +268,9 @@ hosts:
         intel-ubuntu2204-x64-1:
             ip: 92.51.196.114
             build_test_v8: yes
-            is_benchmark: yes
```

Comment:
I've removed this because I'm not sure if we need this set (the other benchmark machine doesn't set this) and the role is currently broken -- it tries to install packages from [`ansible/roles/benchmarking/vars/main.yml`](https://github.com/nodejs/build/blob/f45d15732a8470f661f89122ad87dc76130aeea3/ansible/roles/benchmarking/vars/main.yml) and fails to find https://github.com/nodejs/build/blob/f45d15732a8470f661f89122ad87dc76130aeea3/ansible/roles/benchmarking/vars/main.yml#L11-L13. Presumably these are available package differences after the upgrade to Ubuntu 22.04.